keyboards/crkbd/keymaps/drashna/keymap.c

```diff
@@ -131,9 +131,9 @@ bool process_record_keymap(uint16_t keycode, keyrecord_t *record) {
 oled_rotation_t oled_init_user(oled_rotation_t rotation) { return OLED_ROTATION_270; }
 
 void add_keylog(uint16_t keycode) {
-    if ((keycode >= QK_MOD_TAP && keycode <= QK_MOD_TAP_MAX) || (keycode >= QK_LAYER_TAP && keycode <= QK_LAYER_TAP_MAX)) {
-        keycode = keycode & 0xFF;
-    }
+//    if ((keycode >= QK_MOD_TAP && keycode <= QK_MOD_TAP_MAX) || (keycode >= QK_LAYER_TAP && keycode <= QK_LAYER_TAP_MAX)) {
```

Comment:
Could you revert this change to my keymap?

Reply:
Honestly don't even know how this made it in! my bad. will revert

Reply:
should be reverted to your master